admin/app/controllers/solidus_admin/orders_controller.rb

```diff
@@ -17,5 +17,19 @@ def index
         format.html { render component('orders/index').new(page: @page) }
       end
     end
+
+    def new
+      @order = Spree::Order.new(
+        {
+          created_by: current_solidus_admin_user,
+          frontend_viewable: false,
+          store_id: current_store.try(:id)
+        }
```

Comment:
nit: don't need to wrap in curly braces, I would even expected rubocop to complain abou this 🤷

Reply:
Good catch! You're right, it's a copy-paste leftover from[ the old backend orders controller](https://github.com/solidusio/solidus/blob/main/backend/app/controllers/spree/admin/orders_controller.rb#L162-L166) 😅  Will clean that up, thanks!